manon/language-selector-list-variables.scss

```diff
@@ -55,10 +55,10 @@
   );
 
   /* List opened button icon */
-  --language-selector-list-open-button-icon: "\e901";
+  --language-selector-list-open-button-icon: "⯆";
```

Comment:
It is intended to clear the default values for the icons?

Reply:
The intent here is to set a fallback icon in case the user doesn't pick a theme or the theme somehow can't be imported.

By "fallback icon" what we really mean is a fallback glyph, but in reality we're setting a code point. And which code point results in which glyph entirely depends on the theme (and its icon font).

So as a fallback, as long as `--icon-font-family` is unset, I think it makes sense to use code points which correspond to a useful standard utf8 glyph.